_posts/2020-01-20-libmemkind.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Introduction to libmemkind
+author: jschmieg
+layout: post
+identifier: libmemkind
+---
+
+
+# Introduction
+
+One of the components of PMDK is libmemkind, a library which was developed to
```

Comment:
1) memkind is not part of PMDK, no matter how crazy logic you will apply
2) memkind was not created for persistent memory, it was updated to support it

Reply:
done